pyxrf/model/fileio.py

```diff
@@ -88,12 +91,68 @@ def __init__(self, **kwargs):
         self.mask_data = None
 
         # Display PyXRF version in the window title
-        global pyxrf_version
-        if pyxrf_version[0] not in 'vV':
-            pyxrf_version = f"v{pyxrf_version}"
-        self.window_title_base = f"PyXRF: X-ray Fluorescence Analysis Tool ({pyxrf_version})"
+        ver, new_ver = self._get_pyxrf_version_str()
+        ver_str = ver
+        if new_ver is not None:
+            ver_str += f" - new version {new_ver} is available"
+        self.window_title_base = f"PyXRF: X-ray Fluorescence Analysis Tool ({ver_str})"
         self.window_title = self.window_title_base
 
+    def _get_pyxrf_version_str(self):
+        """
+        The function returns the tuple of strings:
+        - current version number of PyXRF;
+        - the latest version of PyXRF from nsls2forge channel.
+        If current version is the latest, then the second string None.
+        """
+
+        # Determine the current version of PyXRF
+        global pyxrf_version
+        pyxrf_version_str = pyxrf_version
+        if not pyxrf_version_str.lower().startswith('v'):
+            pyxrf_version_str = f"v{pyxrf_version_str}"
+
+        logger.info("Checking for new version availability ...")
+
+        # Now find the latest version available at nsls2forge
+        pyxrf_latest_version_str = None
+        try:
+            # Create a list of available versions
+            r = requests.get('https://conda.anaconda.org/nsls2forge/noarch/repodata.json')
+            pkgs = r.json()
+            pyxrf_ver = []
+            for pkg in pkgs['packages'].keys():
+                if pkg.startswith('pyxrf'):
+                    pkg_version = LooseVersion(pkg.split('-')[1])
+                    pyxrf_ver.append(pkg_version)
+
+            if len(pyxrf_ver):
+                max_version = pyxrf_ver[0]
+                for n, pkg_version in enumerate(pyxrf_ver):
+                    if max_version < pkg_version:
+                        max_version = pkg_version
+
+                current_version = LooseVersion(pyxrf_version)
+
+                if current_version < max_version:
+                    pyxrf_latest_version_str = f"v{max_version}"
+
+            if pyxrf_latest_version_str is not None:
+                logger.info(f"New version of PyXRF ({pyxrf_latest_version_str}) was found "
+                            "at the 'nsls2forge' conda channel")
```

Comment:
Maybe use `in` to be consistent with the message in the `except`?
```suggestion
                            "in the 'nsls2forge' conda channel")
```